### DISCUSSION TITLE
Is it possible to save the mesh from 'Gmsh3D("something.geo")'?

### DISCUSSION THREAD

**Discussion by CalebDmArcher** [usnistgov/fipy#1004] [Q&A]
Hi,

I am using Gmsh to generate the mesh and use
```
...
mesh = Gmsh3D("cube.geo")
phi = CellVariable(name="solution variable", mesh=mesh, value=T_ambient)
...
```
to set up the solution variable.

As my mesh gets finer and finer, it starts to take longer time to generate the mesh so I feel like it is unnecessary to regenerate the mesh.
I noticed that `Gmsh3D` might be a Fipy function. 
Is it possible to save the results of it somewhere so I can directly use it next time?

**Comment by guyer** [comment]
When you call `Gmsh3D("cube.geo")`, the first thing it does is invoke the shell command
```bash
gmsh cube.geo -3 -nopopup -format msh2 -smooth 8 -o cube.msh
```
which converts the `.geo` geometry description to a `.msh` mesh definition.
So, you can do that manually, and then call
```python
mesh = Gmsh3D("cube.msh")
```
